### ISSUE TITLE
Incorrect guava version in project created from archetype

### ISSUE THREAD

**Issue by annabudziszewska** [Cognifide/bobcat#198]
After creating new project from AEM BDD Bobcat archetype, guava 15.0 was imported

Workaround: set new guava version in pom file:
```
<dependency>
            <groupId>com.google.guava</groupId>
            <artifactId>guava</artifactId>
            <version>21.0</version>
</dependency>
```

**Comment by mkrzyzanowski** [comment]
This is a part of a bigger cleanup of all project dependencies.

**Comment by mkrzyzanowski** [comment]
Should be resolved with #243
